articles/install-guide/update.md

```diff
@@ -0,0 +1,163 @@
+
+    ```bash
+    Name: qsharp
+    Version: 0.9.1909.3002
+    Summary: Python client for Q#, a domain-specific quantum programming language
+    ...
+    ```
+
+1. You can now use the updated QDK version to run your existing quantum programs.
+
+## Jupyter notebooks
+
+1. Update the `iqsharp` kernel
+
+    ```bash
+    dotnet tool update -g Microsoft.Quantum.IQSharp
+    dotnet iqsharp install
+    ```
+
+1. Verify the `iqsharp` version
+
+    ```bash
+    dotnet iqsharp --version
+    ```
+
+    You should see the following output:
+
+    ```bash
+    iqsharp: 0.9.1909.3002
+    Jupyter Core: 1.1.14623.0
+    ```
+
+1. You can now open an existing Jupyter notebook and run it with the updated QDK.
+
+## C# on Windows, using Visual Studio
+
+1. Update the Q# Visual Studio extension
+
+    - Visual Studio prompts you to accept updates to the [Quantum Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+    - Accept the update
+
+    > [!NOTE]
+    > The project templates are updated with the extension. The updated templates apply to newly created projects only. The code for your existing projects is not updated when the extension is updated.
+
+1. Update the QDK packages
+
+    - Open an existing application
+    - Select **Dependencies** in the Solution Explorer
+    - Select **Manage NuGet Packages**
+    - Update the **Microsoft.Quantum** packages to the latest version
+
+1. You can now run your application with the latest QDK
+
+## C#, using VS Code
+
+1. Update the Quantum VS Code extension
+
+    - Restart VS Code
+    - Navigate to the **Extensions** tab
+    - Select the **Microsoft Quantum Development Kit for Visual Studio Code** extension
+    - Reload the extension
+
+1. Update the Quantum project templates:
+
+   - Go to **View** -> **Command Palette**
+   - Select **Q#: Install project templates**
+
+1. Open an existing application in VS Code
+
+   - Edit the .csproj file to add the new package versions
+
+    ```xml
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Quantum.Standard" Version="0.9.1909.3002" />
+        <PackageReference Include="Microsoft.Quantum.Development.Kit" Version="0.9.1909.3002" />
+    </ItemGroup>
+    ```
+
+1. You can now run your application with the updated QDK
+
+## C#, using the `dotnet` command-line tool
+
+1. Update the Quantum project templates for .NET
+
+    ```bash
+    dotnet new -i Microsoft.Quantum.ProjectTemplates
+    ```
+
+    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
+
+1. Update and run an existing application
+
+    - Update the QDK package versions in your application
+
+        ```bash
+        dotnet add package Microsoft.Quantum.Development.Kit
+        dotnet add package Microsoft.Quantum.Standard
+        ```
+
+    - Run the application
+
+        ```bash
+        dotnet run
+        ```
+
+    - Your application will run with the new package versions
+
```

Comment:
Do we want to mention the project update instructions here as well?

Reply:
As discussed offline, the csproj will be updated by the dotnet add package ...

Reply:
Never mind, sorry, I had misread your changes. Looks good!